Main.cpp

```diff
@@ -252,6 +252,34 @@ void _LoadingSplashProgressReport(std::string reportString, bool newItem = false
 
 void LoadingSplashProgressReport(std::string reportString, bool newItem = false)
 {
+	if(g_LogToCli)
+	{
+		if (newItem)
+		{
+			std::cout << std::endl;
+		}
+		// overwrite current line
```

Comment:
This is very minor (especially considering the codebase is kinda messy) but please try to match codebase's commenting style - `// Start comment with capital letter`
So the one above would be `// Overwrite current line`